tests/test_benchmarks_web_urldispatcher.py

```diff
@@ -18,6 +19,20 @@
 from aiohttp.http import HttpVersion, RawRequestMessage
 
 
+@pytest.fixture
+def github_urls() -> list[str]:
+    """GitHub api urls."""
+    # The fixture provides OpenAPI generated info for github.
+    # To update the local data file please run the following command:
+    # $ curl https://raw.githubusercontent.com/github/rest-api-description/refs/heads/main/descriptions/api.github.com/api.github.com.json | jq ".paths | keys" > github-urls.json
+
+    here = Path(__file__).parent
+    with (here / "github-urls.json").open() as f:
+        urls = json.load(f)
+
+    return cast(list[str], urls)
```

Comment:
We shouldn't cast, it bypasses any type safety that may be present and could produce problems later on. Here we can just do:
```
        urls: list[str] = json.load(f)
```